lib/hardware/modules/div/iob_div_subshift_signed/iob_div_subshift_signed.py

```diff
@@ -3,11 +3,219 @@ def setup(py_params_dict):
+                        "width": "DATA_W",
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "done_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "done",
+                        "width": 1,
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "quotient_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "quotient",
+                        "width": "DATA_W",
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "remainder_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "remainder",
+                        "width": "DATA_W",
+                        "direction": "output",
+                    },
+                ],
+            },
+        ],
+        "wires": [
+            {
+                "name": "rq",
+                "descr": "rq wire",
+                "signals": [
+                    {"name": "rq", "width": "2*DATA_W"},
+                ],
+            },
+            {
+                "name": "divisor_reg",
+                "descr": "divisor_reg wire",
+                "signals": [
+                    {"name": "divisor_reg", "width": "DATA_W"},
+                ],
+            },
+            {
+                "name": "divident_sign",
+                "descr": "divident_sign wire",
+                "signals": [
+                    {"name": "divident_sign", "width": 1},
+                ],
+            },
+            {
+                "name": "divisor_sign",
+                "descr": "divisor_sign wire",
+                "signals": [
+                    {"name": "divisor_sign", "width": 1},
+                ],
+            },
+            {
+                "name": "pc",
+                "descr": "pc wire",
+                "signals": [
+                    {"name": "pc", "width": "PC_W"},
+                ],
+            },
+            {
+                "name": "subtraend",
+                "descr": "subtraend wire",
+                "signals": [
+                    {"name": "subtraend", "width": "DATA_W"},
+                ],
+            },
+            {
+                "name": "tmp",
+                "descr": "tmp wire",
+                "signals": [
+                    {"name": "tmp", "width": "DATA_W+1"},
+                ],
+            },
+        ],
+        "snippets": [
+            {
+                "verilog_code": """
+            assign quotient_o  = rq[DATA_W-1:0];
+            assign remainder_o = rq[2*DATA_W-1:DATA_W];
+
+   always @(posedge clk_i) begin
```

Comment:
In the future we will want to replace all `always @(posedge clk_i)` by `always @*`.
@jjts Should the conversion of this `always` block be done in this PR?

Reply:
Related to issue https://github.com/IObundle/iob-soc/issues/762